Harmony/Tools/AccessTools.cs

```diff
@@ -1395,6 +1407,58 @@ public static bool IsOfNullableType<T>(T instance)
 			return Nullable.GetUnderlyingType(typeof(T)) != null;
 		}
 
+		/// <summary>Tests whether a type or member is static, as defined in C#</summary>
+		/// <param name="member">type or member</param>
+		/// <returns>True if the type or member is static</returns>
+		///
+		public static bool IsStatic(MemberInfo member)
```

Comment:
Maybe add null check and make it an extension method?

Reply:
Added `null` check. I thought about making it an extension method, but it would look awkward with the existing `IsStatic` property on `MethodBase` and `FieldInfo`. If it were possible to add extension properties, I would have gone with that approach.